src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -217,3 +217,34 @@ import { ViewTransitions } from 'astro:transitions';
 :::note[known limitations]
 The `morph` animation cannot be simulated in traditional CSS. So any element using this animation will not be animated.
 :::
+
+## Caveats
+
+When using the `<ViewTransitions />` component page navigations do not behave exactly the same as they do when navigating directly to a page (such as typing the URL into the address bar).
```

Comment:
```suggestion
`<ViewTransitions />` is a routing component that affects how scripts are executed during page navigation.
```
Related to comment above, would something like this be an accurate introduction to the points below?